### ISSUE TITLE
[Question] Block the exp reward for players under a limit of level

### ISSUE THREAD

**Issue by Tiplish** [Eniripsa96/SkillAPI#358]
Hi. 

I'm using the compatibility of mythicmob and skillapi to give reward to players. But i would like to block this exp reward if the player is under or upper a limit of level. There is a way already implemented?

Thanks

**Comment by Eniripsa96** [comment]
SkillAPI doesn't have this functionality. Where do the limits come from? Mob level?

**Comment by Tiplish** [comment]
Yes, we will have Mythicmobs who are defined to be only for players around a level (20 for example). Every player who are under level 20 have to don't get exp when they killed this mob.

Mobs level 20 give more exp than mobs level 1 and so, i don't want that player could farm high level mobs to get a faster leveling phase.
